lib/features/digital_guide_view/presentation/digital_guide_view.dart

```diff
@@ -0,0 +1,118 @@
+import "../data/repository/image_repository.dart";
+import "widgets/accessibility_button.dart";
+import "widgets/digital_guide_data_source_link.dart";
+import "widgets/digital_guide_features_section.dart";
+import "widgets/headlines_section.dart";
+
+@RoutePage()
+class DigitalGuideView extends ConsumerWidget {
+  const DigitalGuideView({
+    @PathParam("id") required this.id,
+  });
+
+  final int id;
+
+  @override
+  Widget build(BuildContext context, WidgetRef ref) {
+    final asyncDigitalGuideData = ref.watch(getDigitalGuideDataProvider(id));
+    // question: Should the app bar appear during loading or when there's an error?
+    // Now it doesn't, neither does it appear on SKS menu screen
+    return asyncDigitalGuideData.when(
+      data: _DigitalGuideView.new,
+      // To-do nice error
+      error: (error, stackTrace) => Text("API error occured: $error"),
+      loading: () => const Scaffold(
+        body: Center(
+          child: CircularProgressIndicator(),
+        ),
+      ),
+    );
+  }
+}
+
+class _DigitalGuideView extends ConsumerWidget {
+  const _DigitalGuideView(this.digitalGuideResponse);
+
+  final DigitalGuideResponse digitalGuideResponse;
+
+  @override
+  Widget build(BuildContext context, WidgetRef ref) {
+    final asyncImageResponse =
+        ref.watch(getImageDataProvider(digitalGuideResponse.images[0]));
+
+    return Scaffold(
+      appBar: DetailViewAppBar(
+        title: context.localize.map,
+        actions: [
+          AccessibilityButton(),
+        ],
+      ),
+      body: CustomScrollView(
+        slivers: [
+          SliverList(
+            delegate: SliverChildListDelegate([
+              const SizedBox(height: DigitalGuideConfig.heightSmall),
+              asyncImageResponse.when(
+                data: (imageResponseData) => Image.network(
+                  imageResponseData.imageUrl,
+                  fit: BoxFit.fitWidth,
+                ),
+                loading: () => const Center(child: CircularProgressIndicator()),
+                // To-do nice error
+                error: (error, stack) => Text("Error loading images: $error"),
+              ),
+              HeadlinesSection(
+                name: digitalGuideResponse.translations.pl.name,
+                description: digitalGuideResponse.translations.pl.extendedName,
+              ),
+              ContactSection(
+                list: IList<ContactIconsModel>([
+                  ContactIconsModel(
+                    text: digitalGuideResponse.translations.pl.address
+                        .replaceAll("ulica", "ul."),
+                    icon: Assets.svg.contactIcons.compass,
+                  ),
+                  ContactIconsModel(
+                    text: digitalGuideResponse.telephoneNumber
+                        .toPolishPohoneNumber(),
+                    icon: Assets.svg.contactIcons.phone,
+                    // To-do url not working, nothing happens
+                    url:
```

Comment:
Please if needed make it as en extension but we should probably format that property inside repository

Reply:
@mikolaj-jalocha Can you please provide clarification or explanation to this?

Reply:
Formatting phone number (reducing <p> marks etc.) should be done in the repository.